components/Policies/PolicyPage.module.css

```diff
@@ -3,55 +3,71 @@
   font-family: "Nunito";
 }
 
-.subHeading {
-  font-size: 1.5rem;
-  font-weight: bold;
-  margin-top: 0;
+.sharedValues {
+  font-size: 1rem;
   margin-left: 0.2rem;
+  letter-spacing: 0.015em;
+  text-align: justify;
+  font-style: normal;
+  font-weight: 700;
+  color: #1a3185;
+  line-height: 125%;
+  margin-top: 1rem;
   margin-bottom: 0.4rem;
-  letter-spacing: -0.625px;
-  text-align: center;
-  width: 100%;
 }
 
-.subHeading2 {
-  font-size: 1rem;
+.subHeading {
+  font-weight: bold;
   margin-top: 0;
-  margin-left: 0.2rem;
-  margin-bottom: 0.4rem;
-  letter-spacing: -0.625px;
-  text-align: center;
-  width: 100%;
 }
 
-.subHeading2Underlined {
-  font-size: 1rem;
-  text-decoration: underline;
-  margin-top: 0;
-  margin-left: 0.2rem;
-  margin-bottom: 0.4rem;
-  letter-spacing: -0.625px;
-  text-align: center;
+.text1 {
+  font-weight: 400;
+}
+
+.text2 {
+  margin-bottom: 1.5rem;
+}
+
+.boxContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
   width: 100%;
+  height: 102.73px;
```

Comment:
I'm curious where this number is coming from?

Reply:
So I copy and pasted those values from figma. Are you saying it is better to keep it at a whole number?

Reply:
Ah I suppose that's a good reason... I would personally round to 100px to make it less of a "magic" number, but it seems reasonable to take a constant number like that from the designs.

In general, I'm usually asking myself when creating CSS sizing rules "how can I make sure that the way this value is calculated makes sense for all sorts of screen dimensions?" If there's a more "proportional" or "scalable" unit to use, e.g. ["em" or "rem"](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#relative_length_units), I might prefer those units. This doesn't seem to be a place where that applies though.